apps/emqx_auth_ext/test/emqx_auth_ext_test_tls_certs_helper.erl

```diff
@@ -0,0 +1,307 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2024 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%--------------------------------------------------------------------
+
+-module(emqx_auth_ext_test_tls_certs_helper).
```

Comment:
can this module reside in the OSS part?
at least the gen_ functions can be useful there too ?

Reply:
Is there any known method to make this app find the test helper module resides in another app while running common test?

I only know this way: 
https://github.com/emqx/emqtt/blob/master/test/emqtt_test_lib.erl#L58-L70